vault/resource_pki_secret_backend_role_test.go

```diff
@@ -371,6 +371,7 @@ resource "vault_pki_secret_backend_role" "test" {
   basic_constraints_valid_for_non_ca = false
   not_before_duration                = "45m"
   allowed_serial_numbers             = ["*"]
+  cn_validations					 = ["email", "hostname"]
```

Comment:
Can we add assertions that the values are being set properly. Something like this for the initial checks 

```
resource.TestCheckResourceAttr(resourceName, "cn_validations.#", "2"),
resource.TestCheckTypeSetElemAttr(resourceName, "cn_validations.*", "email"),
resource.TestCheckTypeSetElemAttr(resourceName, "cn_validations.*", "hostname"),
```

And this for when we update the role within the test?

```
resource.TestCheckResourceAttr(resourceName, "cn_validations.#", "1"),
resource.TestCheckTypeSetElemAttr(resourceName, "cn_validations.*", "disabled"),
```